api/models/SdtdConfig.js

```diff
@@ -34,6 +34,12 @@ module.exports = {
       defaultsTo: false
     },
 
+    //When a server does not respond to requests for a long time, it is set to inactive.
```

Comment:
I think you copy pasted a bit too much here

Reply:
yep